Find a functionally equivalent code:com.android.systemui.screenshot.GlobalScreenshot.saveScreenshotInWorkerThread:COMMENT
Method Modifier: private     
Comment:/**
 * Creates a new worker thread and saves the screenshot to the media store.
 */

@@ -4,9 +4,11 @@
     data.image = mScreenBitmap;
     data.iconSize = mNotificationIconSize;
     data.finisher = finisher;
+    data.previewWidth = mPreviewWidth;
+    data.previewheight = mPreviewHeight;
     if (mSaveInBgTask != null) {
         mSaveInBgTask.cancel(false);
     }

